### PR TITLE
Implementing new combinators into dblPendulum

### PR DESCRIPTION
**Pull request by Ant13731**
Contributes #2399
Contributes #2476 
Implemented some higher level combinators in the pendulum example. I think `compoundNC` allows for a level 3 combination between `NamedIdeas`, but I had to place it into the database. Although I'm not sure if `Assumptions.hs` is the best place for it.

### CODE REVIEW

File: code/drasil-example/Drasil/DblPendulum/Assumptions.hs [hunk 1]
```diff
@@ -20,18 +22,18 @@ yAxisDir        = cic "yAxisDir"          yAxisDirDesc        "yAxisDir"
 startOrigin     = cic "startOrigin"       startOriginDesc     "startOrigin"     assumpDom
 
 pend2DMotionDesc :: Sentence
-pend2DMotionDesc = S "The" +:+ phrase pendulum +:+ phrase motion `S.sIs` phrase twoD +:+. sParen (getAcc twoD)
+pend2DMotionDesc = atStartNP (the (compoundNC pendulum motion)) `S.sIs` phrase twoD +:+. sParen (getAcc twoD)
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/Assumptions.hs:25]: it's probably not a good idea to use `compoundNC` inline like this. You should give this a Haskell-level name.

File: code/drasil-example/Drasil/DblPendulum/Body.hs [hunk 2]
```diff
@@ -94,12 +94,12 @@ justification :: Sentence
 justification = foldlSent [S "A", phrase pendulum, S "consists" `S.sOf` phrase mass, 
                             S "attached to the end of a", phrase rod `S.andIts` S "moving curve" `S.sIs`
                             (S "highly sensitive to initial conditions" !.), S "Therefore" `sC`
-                            S "it is useful to have a", phrase program, S "to simulate", phrase motion
-                            `S.the_ofThe` phrase pendulum, (S "to exhibit its chaotic characteristics" !.),
+                            S "it is useful to have a", phrase program, S "to simulate", phraseNP (motion
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/Body.hs:97]: here too, this should be named. Same below.

File: code/drasil-example/Drasil/DblPendulum/Concepts.hs [hunk 3]
```diff
@@ -3,9 +3,18 @@ module Drasil.DblPendulum.Concepts where
...
 
+--below imports needed compoundNC part to work
+import Utils.Drasil
+import Data.Drasil.Concepts.Physics (pendulum, motion, position, velocity, force)
+
 
 concepts :: [IdeaDict]
-concepts = nw newtonSLRRC : map nw [rod, horizontal, vertical] ++ map nw defs
+concepts = nw newtonSLRRC : map nw [rod, horizontal, vertical] 
+  ++ map nw defs ++ map nw [compoundNC pendulum motion,
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/Concepts.hs:13]: aha, with all the previous things named, this list would then shorten!

**Review by JacquesCarette — CHANGES_REQUESTED** [review]
Overall, good stuff. But as this is being edited, may as well do it 'right'.  Nice side-effect of fixing bad periods on the output.

**Comment by Ant13731** [comment]
This should be okay to merge now (and resolve the requested changes). I know we already discussed changing the naming scheme and moving some combinators in other issues, but I think it would be easier to just start on a fresh branch.

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**